### PR TITLE
Show events for pods when viewing pod-details.

### PR DESCRIPTION
**Pull request by kenan435**
Add a testing method for showing events when in pod-details

### CODE REVIEW

**Comment by bryk** [comment]
I like this! @floreks @maciaszczykm Can you review?

File: src/app/backend/resource/pod/detail.go [hunk 1]
```diff
@@ -79,11 +79,11 @@ type Controller struct {
...
-	JobList                   *joblist.JobList                                     `json:"joblist,omitempty"`
-	ReplicaSetList            *replicasetlist.ReplicaSetList                       `json:"replicasetlist,omitempty"`
-	ReplicationControllerList *replicationcontrollerlist.ReplicationControllerList `json:"replicationcontrollerlist,omitempty"`
-	DaemonSetList             *daemonsetlist.DaemonSetList                         `json:"daemonsetlist,omitempty"`
-	StatefulSetList           *statefulsetlist.StatefulSetList                     `json:"statefulsetlist,omitempty"`
+	JobList                   *joblist.JobList                                     `json:"job,omitempty"`
+	ReplicaSetList            *replicasetlist.ReplicaSetList                       `json:"replicaset,omitempty"`
+	ReplicationControllerList *replicationcontrollerlist.ReplicationControllerList `json:"replicationcontroller,omitempty"`
+	DaemonSetList             *daemonsetlist.DaemonSetList                         `json:"daemonset,omitempty"`
+	StatefulSetList           *statefulsetlist.StatefulSetList                     `json:"statefulset,omitempty"`
```
Comment (maciaszczykm) [inline src/app/backend/resource/pod/detail.go]: It was okay before, please undo these changes.

File: src/app/backend/handler/apihandler.go [hunk 2]
```diff
@@ -1407,7 +1407,7 @@ func (apiHandler *APIHandler) handleGetThirdPartyResource(request *restful.Reque
 func (apiHandler *APIHandler) handleGetThirdPartyResourceDetail(request *restful.Request,
 	response *restful.Response) {
 	name := request.PathParameter("thirdpartyresource")
-	result, err := thirdpartyresource.GetThirdPartyResourceDetail(apiHandler.client, apiHandler.clientConfig, name)
```
Comment (maciaszczykm) [inline src/app/backend/handler/apihandler.go]: It was okay before, please undo these changes otherwise it won't pass CI.

File: src/app/backend/resource/pod/events.go [hunk 3]
```diff
@@ -7,11 +7,10 @@ import (
 	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/event"
 	client "k8s.io/client-go/kubernetes"
-	api "k8s.io/client-go/pkg/api/v1"
 )
 
 // Get all events that are associated with this pod.
-func GetEventsForPods(client client.Interface, dsQuery *dataselect.DataSelectQuery, namespace, podName string) (
+func GetEventsForPod(client client.Interface, dsQuery *dataselect.DataSelectQuery, namespace, podName string) (
```
Comment (maciaszczykm) [inline src/app/backend/resource/pod/events.go]: Comments for backend methods should start with methods names to fulfill style requirements: `GetEventsForPod gets...`. We should comment all public methods.

**Comment by maciaszczykm** [comment]
Please also make sure, that you format Go files properly. Testing method looks good.

**Comment by maciaszczykm** [comment]
LGTM, merging.
